config/config.go

```diff
@@ -178,28 +178,26 @@ func (c *CombinerConfig) initComponent() (fiber.Component, error) {
 // ProxyConfig is used to parse the configuration for a Proxy
 type ProxyConfig struct {
 	ComponentConfig
-	Endpoint string   `json:"endpoint" required:"true"`
-	Timeout  Duration `json:"timeout"`
-	Protocol string   `json:"protocol"`
+	Endpoint string            `json:"endpoint" required:"true"`
+	Timeout  Duration          `json:"timeout"`
+	Protocol protocol.Protocol `json:"protocol"`
 	GrpcConfig
 }
 
 type GrpcConfig struct {
-	Service string `json:"service,omitempty"`
-	Method  string `json:"method,omitempty"`
+	ServiceMethod string `json:"service_method,omitempty"`
```

Comment:
No objection with the change. But could there be any scenario in the future where the two are required separately? If so, I feel that it could be useful to keep them separate and add a utility method to get the concatenated value:

```golang
func (cfg * GrpcConfig) ServiceMethod() string {
    return fmt.Sprintf("%s/%s", cfg.Service, cfg.Method)
}
```

If this is not a possibility, please ignore the suggestion.

Reply:
It is possible scenario is whereby we uses multiple method from a single service, however for the current implementation, each dispatcher/caller should and can only call one endpoint. 

Since connection (tied to the method) are created at dispatcher init and reused (for optimisation), it don't support any method as well, as such they are always coupled together based on our implementation. 

Even if a service has multiple endpoint, the fiber way to do it would be treat them as different routes, as such its 1:1 mapping.